cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/template/etl/common/DefaultPipelineConfigurer.java

```diff
@@ -109,6 +109,10 @@ public void createDataset(String datasetName, Class<? extends Dataset> datasetCl
   }
   
   private String getPluginId(String childPluginId) {
+    // for configuring transform plugin types we don't want to include pluginPrefix
```

Comment:
shouldn't ignore the prefixing. This is done to provide isolation across plugins that use plugins.  For example, your validator transform can use a plugin called 'core' and some other random transform can use a plugin called 'core' without them stepping on each other.